master/reference/calicoctl/resources/ippool.md

```diff
@@ -40,11 +41,7 @@ spec:
 | ipipMode | The IPIP mode defining when IPIP will be used. | Always, CrossSubnet, Never | string| `Never` |
 | natOutgoing | When enabled, packets sent from {{site.prodname}} networked containers in this pool to destinations outside of this pool will be masqueraded. | true, false | boolean | `false` |
 | disabled | When set to true, {{site.prodname}} IPAM will not assign addresses from this pool. | true, false | boolean | `false` |
-
-> **Important**: Do not use a custom `blockSize` until **all** Calico components have been updated to a version that 
```

Comment:
Are we not keeping this note? It feels like an important one to keep, at least for a few more releases.

Reply:
Generally better to keep the contents of the v3.5 directory confined to the v3.5 release. That is the release they are on and reading about. It can cause confusion to discuss other releases.